freqtrade/wallets.py

```diff
@@ -214,6 +214,18 @@ def get_total_stake_amount(self):
                                 self._config['tradable_balance_ratio'])
         return available_amount
 
+    def get_reserved_stake_amount(self):
+        """
+        Return the total currently reserved balance in stake currency.
+        Calculated as
+        (<open_trade stakes> + free amount) - tot_profit - available_capital
+        """
+        available_capital = self._config.get('available_capital') or 0
+        tot_profit = Trade.get_total_closed_profit()
+        open_stakes = Trade.total_open_trades_stakes()
+        available_balance = self.get_free(self._config['stake_currency'])
+        return available_balance + open_stakes - tot_profit - available_capital
```

Comment:
This function is only covering half of the options.

There's 2 possibilities to limit the used capital - either via `available_capital`, or via `tradable_balance_ratio`.
You can have a look at the logic in `get_total_stake_amount()` (right above this function).